### PR TITLE
Fix xfs

### PR DESCRIPTION
**Pull request by PraveenPenguin**
1-Fix compilation issue in recent ubuntu distro 
2- fixed multiple pep8 issue

### CODE REVIEW

File: fs/xfstests.py [hunk 1]
```diff
@@ -65,11 +65,14 @@ def setUp(self):
                 packages.extend(['libtool'])
             else:
                 packages.extend(['libtool-bin'])
+            if '18' in self.detected_distro.version:
```
Comment (harish-24) [inline fs/xfstests.py]: Can we add this as an `elif` prior to this so that on every run `version` is checked only once?

Comment (PraveenPenguin) [reply fs/xfstests.py]: @harish-24 yes make sense will do as one info it new change in xfs test may for other distro case we also need to take care in future till it clear all other distro

**Comment by PraveenPenguin** [comment]
@harish-24 adopted the changes please have look

**Comment by PraveenPenguin** [comment]
@narasimhan-v can you please have a look on this PR

**Pull request merged**